packages/web-react/src/components/Dropdown/README.md

```diff
@@ -39,7 +40,94 @@ import { Dropdown } from '@lmc-eu/spirit-web-react/components';
 | `trigger.ref`              | `LegacyRef<HTMLButtonElement & HTMLAnchorElement>` | Trigger reference       |
 
+---
+
+# DropdownModern
+
+⚠️ `DropdownModern` component will be deprecated and renamed to `Dropdown` in the next major version.
+This is the React implementation of the [DropdownModern] component.
+
+## Usage
+
+```jsx
+import { DropdownModern, DropdownModernTrigger, DropdownModernPopover } from '@lmc-eu/spirit-web-react/components';
+```
+
+```jsx
+const [isOpen, setIsOpen] = React.useState(false);
+const onToggle = () => setIsOpen(!isOpen);
+
+<DropdownModern id="DropdownExample" isOpen={isOpen} onToggle={onToggle}>
+  <DropdownModernTrigger elementType="button">Trigger button</DropdownModernTrigger>
+  <DropdownModernPopover>…</DropdownModernPopover>
+</DropdownModern>;
+```
+
+### Uncontrolled dropdown
+
+```jsx
+import {
+  UncontrolledDropdown,
+  DropdownModernTrigger,
+  DropdownModernPopover,
+} from '@lmc-eu/spirit-web-react/components';
+```
+
+```jsx
+<UncontrolledDropdown id="UncontrolledDropdownExample">
+  <DropdownModernTrigger elementType="button">Trigger button</DropdownModernTrigger>
+  <DropdownModernPopover>…</DropdownModernPopover>
+</UncontrolledDropdown>
+```
+
+## API
+
+### DropdownModern
+
+| Name               | Type                                             | Default       | Required | Description                                    |
+| ------------------ | ------------------------------------------------ | ------------- | -------- | ---------------------------------------------- |
+| `enableAutoClose`  | `bool`                                           | `true`        | ✕        | Enables close on click outside of Dropdown     |
+| `fullWidthMode`    | [`DropdownFullwidthMode`][dropdownfullwidthmode] | `off`         | ✕        | Full-width mode                                |
+| `id`               | `string`                                         | -             | ✔        | Component id                                   |
+| `isOpen`           | `bool`                                           | `false`       | ✔        | Open state                                     |
+| `onAutoClose`      | `(event: Event) => void`                         | —             | ✕        | Callback on close on click outside of Dropdown |
+| `onToggle`         | `() => void`                                     | —             | ✔        | Function for toggle open state of dropdown     |
+| `placement`        | [Placement dictionary][dictionary-placement]     | `bottom-left` | ✕        | Alignment of the component                     |
+| `UNSAFE_className` | `string`                                         | —             | ✕        | Wrapper custom classname                       |
+| `UNSAFE_style`     | `CSSProperties`                                  | —             | ✕        | Wrapper custom style                           |
+
+### DropdownModernTrigger
```

Comment:
oh, now I see, there is no DropdownModernTrigger? Only DropdownTrigger. Can you fix it everywhere? And the same applies to DropdownModernPopover

Reply:
fixed